scripts/get_umf_debug_info.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+BINARY=$1
+
+if ! which strings >/dev/null; then
+	echo "strings command not found. Please install the binutils package."
+	exit 1
+fi
+
+if [ "$BINARY" = "" ]; then
+	echo "Usage: $(basename $0) <binary_name>"
+	exit 1
+fi
+
+# check if the binary is statically linked with libumf
+if [ $(strings $BINARY | grep "@(#) Intel(R) UMF" | wc -l) -gt 0 ]; then
```

Comment:
`grep -c` does the same thing as piping the output to `wc -l`, same below.

Reply:
Done